src/main/java/bio/terra/workspace/app/controller/WorkspaceApiController.java

```diff
@@ -161,19 +166,22 @@ private ApiWorkspaceDescription buildWorkspaceDescription(Workspace workspace) {
     AuthenticatedUserRequest userReq = getAuthenticatedInfo();
     logger.info("Getting workspace {} for {}", id, userReq.getEmail());
     Workspace workspace = workspaceService.getWorkspace(id, userReq);
-    ApiGcpContext gcpContext =
-        workspace.getGcpCloudContext().map(GcpCloudContext::toApi).orElse(null);
+    ApiWorkspaceDescription desc = buildWorkspaceDescription(workspace);
+    logger.info("Got workspace {} for {}", desc, userReq.getEmail());
 
-    // Note projectId will be null here if no GCP cloud context exists.
-    // When we have another cloud context, we will need to do a similar retrieval for it.
-    ApiWorkspaceDescription desc =
-        new ApiWorkspaceDescription()
-            .id(workspace.getWorkspaceId())
-            .spendProfile(workspace.getSpendProfileId().map(SpendProfileId::id).orElse(null))
-            .stage(workspace.getWorkspaceStage().toApiModel())
-            .gcpContext(gcpContext);
+    return new ResponseEntity<>(desc, HttpStatus.OK);
+  }
 
-    logger.info("Got workspace {} for {}", desc, userReq.getEmail());
+  @Override
+  public ResponseEntity<ApiWorkspaceDescription> updateWorkspace(
+      @PathVariable("workspaceId") UUID id, @RequestBody ApiUpdateWorkspaceRequestBody body) {
```

Comment:
nit: first parameter should be `workspaceId` to match the underlying interface. 
I think the parameter name was changed without updating existing controller methods, so there are probably inconsistencies elsewhere in this file

Reply:
Yes, I think I did the openApi change, but didn't change the code.

Reply:
Done